identity_domain/src/commonMain/kotlin/net/thechance/mena/identity/domain/useCase/validation/mobileNumber/PasswordValidator.kt

```diff
@@ -7,6 +7,9 @@ class PasswordValidator {
                 password.any { it.isUpperCase() }
     }
 
+    fun isPasswordMatch(password: String, confirmPassword: String) =
```

Comment:
Just a small change. Since it's a compact function, it's cleaner to keep it on one line.
```suggestion
    fun isPasswordMatch(password: String, confirmPassword: String) = password == confirmPassword
```